applets/clock/clock.c

```diff
@@ -987,6 +996,80 @@ position_calendar_popup (ClockData *cd)
 
         gtk_window_move (GTK_WINDOW (cd->calendar_popup), x, y);
         gtk_window_set_gravity (GTK_WINDOW (cd->calendar_popup), gravity);
+    }
+#endif
+    /*Only build wayland support when building in process*/
+#ifdef CLOCK_INPROCESS
+#ifdef HAVE_WAYLAND
+
```

Comment:
nitpick: this blank line should be between the X11/Wayland blocks, e.g. be line 1001